src/main/kotlin/tech/thatgravyboat/craftify/ui/UIControls.kt

```diff
@@ -30,65 +31,59 @@ class UIControls : UIContainer() {
     private val position = "https://i.imgur.com/XZWUSSe.png"
     private val settings = "https://i.imgur.com/Nd4gQzY.png"
 
-    private val positionButton = UIButton(URL(position), URL(position), click = {
+    private val positionButton by UIButton(URL(position), URL(position), click = {
```

Comment:
These are changed to by as per [elementa styleguide](https://github.com/EssentialGG/Elementa/blob/master/docs/style-guide.md#language)